airbyte-webapp/src/pages/SettingsPage/pages/DbMigrationPage/components/MigrationComponent.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Could you please also update imports here?
- lib imports
- absolute imports ( `components`, `core` and etc )
- relative imports ( starts with `./` )